tests/Makefile

```diff
@@ -11,7 +11,7 @@ TEST_OBJ = carma_bcoctest.o carma_bc2gtest.o carma_coagtest.o carma_drydeptest.o
 carma_growtest.o carma_inittest.o carma_kappawetrtest.o carma_mietest.o carma_nuctest.o \
 carma_pheattest.o carma_sigmadrydeptest.o carma_sigmafalltest.o carma_swelltest.o \
 carma_vdiftest.o carma_test.o atmosphere_mod.o carma_scfalltest.o carma_growsubtest.o \
-carma_growintest.o carma_fractalmicrotest.o carma_fractalopticstest.o carma_sulfhettests.o
+carma_growintest.o carma_fractalmicrotest.o carma_fractalopticstest.o carma_sulfhettest.o
```

Comment:
Don't you also need to add the 2 new tests here?

Reply:
fixed